### ISSUE TITLE
Multiple images by msg.payload not supported?

### ISSUE THREAD

**Issue by frabnet** [notify-events/node-red#1]
Hi, I'm joining two payloads containing a buffer (a jpg image), so only one message is sent, and it contains msg.payload.img1 and msg.payload.img2.

![buffer](https://user-images.githubusercontent.com/26867706/114733468-a74ad100-9d43-11eb-9154-44065d17dffc.png)

In the notify.events node, I set up two images, one set with Content to msg.payload.img1 and the other pointing to msg.payload.img2.

I would expect a notification with two images, but instead I get this error from the notify node:
**InvalidArgumentError: invalid file**

Is this an unsupported operation? Is there any other way I can achieve what I'm trying to do?

Here is my flow:

![flows](https://user-images.githubusercontent.com/26867706/114734367-6f905900-9d44-11eb-8549-35d68a117d63.png)

Edit: Maybe your node expect just a _buffer_, but I'm passing a _object_. So this I think this is becoming a feature request.

**Comment by matrozov** [comment]
Hello @frabnet.

Indeed, we found that notify.events node does not support nested fields. That means, we could work with **msg.payload**, but not with **msg.payload.img1**. We have fixed this issue in version 1.0.4.
Аlso we found that we had a bug in the code of the related nodejs module witch caused the impossibility of sending a file content as a string. We fixed this as well.

Thanks for the report!
